.travis.yml

```diff
@@ -9,8 +9,8 @@
 
 language: node_js
 os:
-  - windows
   - linux
+  - windows
```

Comment:
Could we keep windows off fro now? Windows support is scheduled for H2.
And I think it would seriously make life easier if currently we share a common travis config.

Reply:
Sure. Let's confirm this decision with the team over slack.